packages/firebase/index.d.ts

```diff
@@ -9976,6 +9984,81 @@ declare namespace firebase.firestore {
     name: string;
     stack?: string;
   }
+
+  type FirebaseSignInProvider =
+    | 'custom'
+    | 'email'
+    | 'password'
+    | 'phone'
+    | 'anonymous'
+    | 'google.com'
+    | 'facebook.com'
+    | 'github.com'
+    | 'twitter.com'
+    | 'microsoft.com'
+    | 'apple.com';
+
+  interface FirebaseIdToken {
+    // Always set to https://securetoken.google.com/PROJECT_ID
```

Comment:
Sorry, one last comment I hope, these comments have to be jsdoc style eg
```
  /**
     * Always set to https://securetoken.google.com/PROJECT_ID
     */
```
or they won't get picked up by the doc generator. If this is supposed to be in the documentation.

Reply:
Done.